libkbfs/folder_branch_ops.go

```diff
@@ -46,6 +46,9 @@ const (
 
 // Constants used in this file.  TODO: Make these configurable?
 const (
+	// The maximum block size in bytes
+	MaxBlockSize = 512 << 10
```

Comment:
I would call this `MaxBlockSizeBytesDefault`, I think.  When we eventually make a DSL engine for the docker tests, we're going to need the ability to set the block size on the command line or via an environment variable (we already do that now in other DSL tests via the config object), so this constant just acts as a default.  And put "Bytes" in the name to clarify the units.